instance/models/instance.py

```diff
@@ -586,4 +697,41 @@ def provision(self):
 
         return (server, ansible_log)
 
+    def provision_external_databases(self):
+        """
+        Provision external databases for this instance
+        """
+        self.provision_external_mysql()
+        self.provision_external_mongo()
+
+    def provision_external_mysql(self):
+        """
+        Provision external mysql database for this instance
+        """
+        if settings.INSTANCE_MYSQL_URL_OBJ:
+            connection = mysql.connect(
+                host=settings.INSTANCE_MYSQL_URL_OBJ.hostname,
+                user=settings.INSTANCE_MYSQL_URL_OBJ.username,
+                passwd=settings.INSTANCE_MYSQL_URL_OBJ.password or '',
+                port=settings.INSTANCE_MYSQL_URL_OBJ.port or 3306,
+            )
+            cursor = connection.cursor()
+
+            # We can't use the database name in a parameterized query, the
+            # driver doesn't escape it properly. Se we escape it here instead
+            database_name = connection.escape_string(self.database_name).decode()
+
+            cursor.execute('CREATE DATABASE IF NOT EXISTS {0}'.format(database_name))
```

Comment:
@omarkhan What if the DB exists, but is already used by another instance? Could we check for DB existence before creating it the first time?